common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -339,12 +341,25 @@ public int chunkFetchHandlerThreads() {
       return 0;
     }
     int chunkFetchHandlerThreadsPercent =
-      conf.getInt("spark.shuffle.server.chunkFetchHandlerThreadsPercent", 100);
```

Comment:
what's wrong with the previous code?

Reply:
No need to give a default value here, when it comes to here, the config must be set.

Reply:
What do you mean by `the config must be set`, @xuanyuanking ? What value do you expect by default? Apparently, this is reverting SPARK-25641 without mentioning SPARK-25641. In the PR, only SPARK-24355 is mentioned.
> No need to give a default value here, when it comes to here, the config must be set.

Reply:
because we only call this method if `separateChunkFetchRequest` returns true.

We will see exception if the assumption is broken.

Reply:
In this PR, we make the separate event loop group configurable by checking the config `spark.shuffle.server.chunkFetchHandlerThreadsPercent` is set or not.
> What do you mean by the config must be set

Here the function `chunkFetchHandlerThreads` is only called while the config is set. https://github.com/apache/spark/blob/0fe203e7032a326ff0c78f6660ab1b09409fe09d/common/network-common/src/main/java/org/apache/spark/network/TransportContext.java#L124-L129

> What value do you expect by default? Apparently, this seems to revert SPARK-25641 together without mentioning SPARK-25641. 

Yes, this PR makes the feature disabled by default, let me also mention SPARK-25641 in PR description.

Reply:
Thank you, @xuanyuanking and @cloud-fan .